pyro/poutine/uncondition_messenger.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import absolute_import, division, print_function
+
+from .messenger import Messenger
+
+
+class UnconditionMessenger(Messenger):
```

Comment:
Could you add a simple test or two for this to `tests/poutine/test_poutines.py`?  It seems like a generally useful utility, and tests help with maintainability.